cli/azd/test/functional/testdata/samples/aspire-full/AspireAzdTests.ServiceDefaults/AspireAzdTests.ServiceDefaults.csproj

```
@@ -12,7 +12,7 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
 
     <PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="8.1.0" />
-    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="8.0.0-preview.3.24076.8" />
+    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="8.0.0-preview.3.24105.21" />
```

YMMV, but if this cannot work without being tied to the same AspireVersion, let's just do...
```suggestion
    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="$(AspireVersion)" />
```